causalpy/pymc_experiments.py

```diff
@@ -336,15 +340,301 @@ def plot(self, counterfactual_label="Counterfactual", round_to=None, **kwargs):
+
+        if correction:
+            _mu_samples_cumulative += results["_systematic_differences"]["cumulative"]
+            _mu_samples_mean += results["_systematic_differences"]["mean"]
+
+        results["ci"] = {
+            "cumulative": [
+                round(np.percentile(_mu_samples_cumulative, ci), 2),
+                round(np.percentile(_mu_samples_cumulative, 100 - ci), 2),
+            ],
+            "mean": [
+                round(np.percentile(_mu_samples_mean, ci), 2),
+                round(np.percentile(_mu_samples_mean, 100 - ci), 2),
+            ],
+        }
+
+        cumulative_upper_mde = round(
+            results["ci"]["cumulative"][1]
+            - results["posterior_estimation"]["cumulative"],
+            2,
+        )
+        cumulative_lower_mde = round(
+            results["posterior_estimation"]["cumulative"]
+            - results["ci"]["cumulative"][0],
+            2,
+        )
+
+        mean_upper_mde = round(
+            results["ci"]["mean"][1] - results["posterior_estimation"]["mean"], 2
+        )
+        mean_lower_mde = round(
+            results["posterior_estimation"]["mean"] - results["ci"]["mean"][0], 2
+        )
+
+        results["posterior_mde"] = {
+            "cumulative": round((cumulative_upper_mde + cumulative_lower_mde) / 2, 2),
+            "mean": round((mean_upper_mde + mean_lower_mde) / 2, 2),
+        }
+        return results
+
+    def power_summary(
+        self, alpha: float = 0.05, correction: bool = False
+    ) -> pd.DataFrame:
+        """
+        Summarize the power estimation results in a DataFrame format.
+
+        This function calls '_power_estimation' to perform power estimation calculations and then
+        converts the resulting dictionary into a pandas DataFrame for easier analysis and visualization.
+
+        Parameters:
+        - alpha (float, optional): The significance level for confidence interval calculations used in power estimation. Default is 0.05.
+        - correction (bool, optional): Indicates whether to apply corrections in the power estimation process. Default is False.
+
+        Returns:
+        - pd.DataFrame: A DataFrame representing the power estimation results, including posterior estimations,
+        systematic differences, confidence intervals, and posterior MDE for cumulative and mean results.
+        """
+        return pd.DataFrame(self._power_estimation(alpha=alpha, correction=correction))
+
+    def power_plot(self, alpha: float = 0.05, correction: bool = False) -> plt.Figure:
+        """
+        Generate and return a figure containing plots that visualize power estimation results.
+
+        This function creates a two-panel plot (for mean and cumulative measures) to visualize the posterior distributions
+        along with the confidence intervals, real mean, and posterior mean values. It allows for adjustments based on
+        systematic differences if the correction is applied.
+
+        Parameters:
```

Comment:
Same comment as above about the parameters and returns blocks not rendering properly in when you build and check the docs locally.